src/Default/default.ts

```diff
@@ -96,7 +96,10 @@ export class DefaultMessageBus implements MessageBus {
                 }
             });
 
-            this.container.globalWindow.addEventListener("message", subscription.listener);
```

Comment:
Added some conditional checks for safety that were more for unit test mocking then production use but good to have regardless.